### PR TITLE
qgis-ltr: 3.28.15 -> 3.34.4

### PR DESCRIPTION
**Pull request by imincik**
## Description of changes

Update QGIS-LTR to the latest version.

Current QGIS-LTR version is continuation of 3.34 version which was the latest version until 3.36 arrived
(see: https://www.qgis.org/en/site/getinvolved/development/roadmap.html)

List of changes between 3.34.3 and 3.34.4 is [following](https://github.com/qgis/QGIS/compare/final-3_34_3%5E...final-3_34_4).

Diff looks big, because of update between 3.28 to 3.34, but diff between 3.34.3 (which we had as the latest version few weeks ago until it was updated to 3.36) is just this small

```diff
--- 	pkgs/applications/gis/qgis/unwrapped-3.34.3.nix  2024-03-15 11:41:47.350360873 +0100
+++ pkgs/applications/gis/qgis/unwrapped-ltr.nix	2024-03-15 11:41:18.353604431 +0100
@@ -77,14 +77,14 @@
     urllib3
   ];
 in mkDerivation rec {
-  version = "3.34.3";
+  version = "3.34.4";
   pname = "qgis-unwrapped";

   src = fetchFromGitHub {
     owner = "qgis";
     repo = "QGIS";
     rev = "final-${lib.replaceStrings [ "." ] [ "_" ] version}";
-    hash = "sha256-uf4qUalY6LxPykgUt/atHBTu+A6ITrcnfez/jwVMLvA=";
+    hash = "sha256-yEltpPhNFT/XB1EB5FvhCKcP0YY4j/q7luhd1mI0ZJU=";
   };

   passthru = {
@@ -97,9 +97,9 @@
     wrapGAppsHook
     wrapQtAppsHook

+    bison
     cmake
     flex
-    bison
     ninja
   ];
```

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

/cc @landam 

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by landam** [comment]
@imincik PR tests locally. Seems to be fine.

File: pkgs/applications/gis/qgis/unwrapped-ltr.nix [hunk 1]
```diff
@@ -76,14 +77,14 @@ let
     urllib3
   ];
 in mkDerivation rec {
-  version = "3.28.15";
-  pname = "qgis-ltr-unwrapped";
+  version = "3.34.4";
+  pname = "qgis-unwrapped";
```
Comment (sikmir) [inline pkgs/applications/gis/qgis/unwrapped-ltr.nix:81]:
```suggestion
  pname = "qgis-ltr-unwrapped";
```

Comment (imincik) [reply pkgs/applications/gis/qgis/unwrapped-ltr.nix:81]: Excellent catch @sikmir !

Comment (imincik) [reply pkgs/applications/gis/qgis/unwrapped-ltr.nix]: Fixed now.

**Pull request merged**